docs/docs/getting_started/quick_start.md

```diff
@@ -106,22 +106,13 @@ Naming can be confusing, specially as you pass them to the `bb` commands. If uns
 The proof is now generated in the `target` folder. To verify it we first need to compute the verification key from the compiled circuit, and use it to verify:
 
 ```sh
-bb write_vk -b ./target/hello_world.json -o ./target/vk
-bb verify -k ./target/vk -p ./target/proof
-```
-
-:::info
-
-Notice that in order to verify a proof, the verifier knows nothing but the circuit, which is compiled and used to generate the verification key. This is obviously quite important: private inputs remain private.
```

Comment:
I removed this section as splitting the proof in UH is more difficult, and I think a bit too much for the quick start guide. The proof can be verified with the verify command as it is anyway. 
Solidity guide still has the script to do the split.

+ This will be improved in next version of bb.js